snippets/logging-alertingrule-app-callouts-snip.adoc

```diff
@@ -0,0 +1,31 @@
+.Example application AlertingRule CR
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: AlertingRule
+  metadata:
+    name: loki-operator-alerts
+    namespace: openshift-operators-redhat <1>
```

Comment:
`openshift-*` namespaces are only for the infrastructure tenant. Only an app namespace can be used: 
(cc @periklis please correct me if I am wrong)

```suggestion
    namespace: app-ns
```